I Love Music/dist/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "author": {
+    "name": "MauriceX",
+    "id": "268768664411439115"
+  },
+  "service": "I Love Music",
+  "description": {
+    "en": "The best Web-Radio of Germany - The best Music for everyone!",
+    "de": "Das beste Internet-Radio in Deutschland - Die beste Musik für jeden!"
+  },
+  "url": "www.ilovemusic.de",
+  "version": "1.0",
+  "logo": "https://i.imgur.com/jO86Aqk.png",
+  "thumbnail": "https://i.imgur.com/LFMmAHc.png",
+  "color": "#F7C516",
+  "tags": ["music","internet-radio","radio"],
+  "category": "music"}
```

Comment:
please put the } on its own line, as its more readable that way xD

Reply:
Alright :D